core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,31 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var answerGroups = responsesService.getAnswerGroups();
+        // Collect all answers which have been handled by at least one
+        // Answer group.
```

Comment:
Thanks! Did you push a new commit?

(Yep, comments do need to begin with a space and a capital letter. But the two lines here are a single comment.)